implementation/contracts/deposit/DepositFunding.sol

```diff
@@ -103,6 +104,14 @@ library DepositFunding {
             block.timestamp > _d.signingGroupRequestedAt.add(TBTCConstants.getSigningGroupFormationTimeout()),
             "Signing group formation timeout not yet elapsed"
         );
+
+        // refund the deposit owner the cost to create a new keep at current price levels.
+        uint256 _seized = _d.seizeSignerBonds();
+
+        /* solium-disable-next-line security/no-send */
+        _d.depositOwner().send(_d.keepSetupFee);
```

Comment:
We should be protected against reentrancy because of our state checks, so this change makes sense. I think I'll pull it to a separate PR though since there are quite a few instances and they all need to be checked against reentrancy.

Reply:
I believe it's gas cost optimization when switching to `call.value`

> Any smart contract that uses transfer() or send() is taking a hard dependency on gas costs by forwarding a fixed amount of gas: 2300.
>
> Our recommendation is to stop using transfer() and send() in your code and switch to using call() instead

https://diligence.consensys.net/blog/2019/09/stop-using-soliditys-transfer-now/

Reply:
I talked to Martin and Alex today, there's a broader recommendation here that we heavily consider withdrawal vs transfer in several cases. Let's talk about it tomorrow.